### PR TITLE
Key action change animation

### PR DESCRIPTION
**Pull request by NejcZdovc**
PR based on #53

### CODE REVIEW

**Comment by NejcZdovc** [comment]
@mondalaci What do you think about the speed of the animation? You can test it with changing keys from Z-M, otherwise popover will block your view.

**Comment by mondalaci** [comment]
@NejcZdovc It think it looks awesome - exactly the way I imagined! Such little changes go a long way regarding user experience.

File: src/components/svg/keys/svg-keyboard-key/svg-keyboard-key.component.ts [hunk 1]
```diff
@@ -44,16 +59,15 @@ export class SvgKeyboardKeyComponent implements OnInit, OnChanges, OnDestroy {
     @Input() height: number;
     @Input() width: number;
     @Input() keyAction: KeyAction;
+    @ViewChild ('rect') rect: ElementRef;
```
Comment (fjozsef) [inline src/components/svg/keys/svg-keyboard-key/svg-keyboard-key.component.ts]: It seems to me it is unused.

Comment (NejcZdovc) [inline src/components/svg/keys/svg-keyboard-key/svg-keyboard-key.component.ts]: Fixed

**Pull request merged**